modules/virt-creating-storage-class.adoc

```diff
@@ -9,6 +9,8 @@
 When you create a storage class, you set parameters that affect the
 dynamic provisioning of persistent volumes (PVs) that belong to that storage class.
 
+In order to use the host path provisioner (HPP) you must create an associated storage class.
+
 [IMPORTANT]
 ====
 When using {VirtProductName} with {product-title} Container Storage, specify RBD block mode persistent volume claims (PVCs) when creating virtual machine disks. With virtual machine disks, RBD block mode volumes are more efficient and provide better performance than Ceph FS or RBD filesystem-mode PVCs.
```

Comment:
This should not be there I think, I think if we want to keep a blurp about rbd, we should say that if you have RBD storage available it is preferable over hpp. That is about it.

Reply:
You'll have to explain this one further to me when we review.